usecases/schema/class.go

```diff
@@ -118,6 +122,9 @@ func (h *Handler) AddClass(ctx context.Context, principal *models.Principal,
 	if err := h.validateCanAddClass(ctx, cls, classGetterWithAuth, false); err != nil {
 		return nil, 0, err
 	}
+	if classHasAsyncReplicationConfigSet(cls) {
```

Comment:
async replication parameters currently set with env variables may also be included as part of the schema. We may allow this as well when the class is created/updated

Reply:
would you be ok with leaving this for a future PR? i'd like to focus on getting the most important things needed for replica movement first and then coming back to finish up moving the rest of the async replication config into this

Reply:
i think i'll also remove the async replication config from the schema.json for now, since we will be updating the target node overrides via raft (also coming in a future PR)